.gitignore

```diff
@@ -2,3 +2,4 @@ build
 llvm-project
 llvm-build
 .vscode
+.cache
```

Comment:
Can you elaborate on what this file or folder does?

Reply:
.cache is the clangd cache folder

Reply:
If we decide not to add this, you could also run this to skip it from appearing in your git status:
```
git update-index --skip-worktree <path_to_cache>
```